man/theme.Rd

```diff
@@ -251,15 +251,20 @@ p1 + theme(
 )
 
 # Axes ----------------------------------------------------------------------
+# Change styles of axes texts and lines
```

Comment:
It seems these diffs of `theme.Rd` come from #2934. It might be better to create a separate PR, but I let this included.

Reply:
Actually, that PR also missed to generate a new vdiffr reference image. Do you have the time to create a separate PR that includes the updated .Rd file and the missing image? That would probably be the best approach.

Reply:
Oh, sure. Will do.